server/classes/root.js

```diff
@@ -0,0 +1,9 @@
+/** The root of the server. All data stored on the server should be (@TODO) accessible from this object */
+export class Root {
+	constructor() {
+		/** @type {Object.<string, import('./player').Player>} */
```

Comment:
Perhaps to make things bit more tidy, we could import types at the top of the file?
```javascript
/** @typedef {import("./player")} Player*/
```

Reply:
good idea

Reply:
done for all files